service/src/chatgpt/index.ts

```diff
@@ -53,8 +53,8 @@ let api: ChatGPTAPI | ChatGPTUnofficialProxyAPI
         options.maxModelTokens = 32768
         options.maxResponseTokens = 8192
       }
-      // if use GPT-4 Turbo
-      else if (/-preview|-turbo/.test(model.toLowerCase())) {
+      // if use GPT-4 Turbo or GPT-4o
+      else if (/-preview|-turbo|-o/.test(model.toLowerCase())) {
```

Comment:
oh, no, it should be `-4o` 🙈

Reply:
Oh I saw that @Chanzhaoyu changed it to just `o`

https://github.com/Chanzhaoyu/chatgpt-web/commit/e3ce91cfa1fb113ae681c6ec8e6ea89f8570b516

Reply:
😁